src/components/NoteListItem/NoteListItem.jsx

```diff
@@ -0,0 +1,92 @@
+import React, { useState, useEffect } from 'react';
+import "./NoteListItem.scss";
+
+export default function NoteListItem({ note, editNotes, localDeleteNote, handleEditNoteText }) {
+
+  const { user, created_at, text } = note;
+  const [editText, setEditText] = useState(false);
+  const [inputText, setInputText] = useState(text);
+
+  useEffect(() => {
+    setEditText(false);
+    setInputText(text);
+  }, [editNotes]);
+
+  const handleEditText = () => {
+    setEditText(true);
+  }
+
+  const handleCancel = () => {
+    setInputText(text);
+    setEditText(false);
+  }
+
+  const localEditNoteText = () => {
+    handleEditNoteText({ ...note, text: inputText });
+    setEditText(false);
+  }
+
+  return (
+    <div className="ticket-card-note-container">
+      <div className="ticket-card-note-header-row">
+        <div className="ticket-card-note-header-row-left">
+          {editNotes ?
+            <div className="ticket-card-note-header-buttons">
+              <button
+                className='mini-rounded'
+                onClick={() => localDeleteNote(note)}>
+                <i className="fas fa-minus icon" />
```

Comment:
Let's switch this out for the "fa-trash" icon

Reply:
Makes perfect sense.